docker-compose.yml

```diff
@@ -8,11 +8,14 @@ services:
     container_name: ${CONTAINER_NAME}
     env_file: mailserver.env
     # To avoid conflicts with yaml base-60 float, DO NOT remove the quotation marks.
+    # More information about the mailserver ports:
+    # https://docker-mailserver.github.io/docker-mailserver/edge/config/security/understanding-the-ports/
     ports:
-      - "25:25"
-      - "143:143"
-      - "587:587"
-      - "993:993"
+      - "25:25"    # SMTP  (explicit TLS => STARTTLS)
+      - "143:143"  # IMAP4 (explicit TLS => STARTTLS)
+      - "465:465"  # ESMTP (implicit TLS)
+      - "587:587"  # ESMTP (explicit TLS => STARTTLS)
+      - "993:993"  # IMAP4 (implicit TLS)
```

Comment:
Very nice change !
That said 3 ports are still missing
```yml
            - "110:110" # POP3
            - "995:995" # POP3
            - "4190:4190" # Sieve filters
```

Reply:
No not really, `docker-compose.yml` is a **not** feature complete example of the mailserver.
Keep in mind that by default POP3 and SIEVE are not enabled https://github.com/docker-mailserver/docker-mailserver/blob/a0f4a375124b3b3402d5639a7d422501c2011b97/mailserver.env#L107-L108 and https://github.com/docker-mailserver/docker-mailserver/blob/a0f4a375124b3b3402d5639a7d422501c2011b97/mailserver.env#L80-L81

Therefore imho it is correct to not list these ports here

Reply:
Yes, but they could be added and commented ;)